lib/rubocop/cop/style/open_ssl_digest_constant.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Algorithmic constants for OpenSSL::Digest are soft-deprecated since
+      # version 2.2.0. Prefer passing a string instead to use hash functions
+      # without defined constants like SHA512-256, SHA3-256 or BLAKE2s256.
+      #
+      # @example instantiation
```

Comment:
`@example` tag is used for `EnforcedStyle`. It is better to use comments for supplementary explanations.

```suggestion
      # @example
      # Below is an example for instantiation.
```